<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRON</class>
 <widget class="QWidget" name="CRON">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="slider_m">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_m">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="slider_h">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_h">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>CRON JOB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Minutes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hours:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
